src/en/howto-privatecloud.md

```diff
@@ -110,7 +119,23 @@ ls ~/simplestreams/*/streams/*
 
 You should see .json files containing the details we just added on the images.
 
-### Upload the Simplestreams Metadata to Swift
+### Use of a local directory for image metadata
+
+Stop here and return to the [bootstrap instructions][bootstrap].
+
+## Upload the image metadata to Swift
+
+!!! Note: Only those with admin privileges or who are operators in the OpenStack 
+environment will be able to create a service and view endpoints used by the following 
+instructions.
+
+### Requirements
+
+ - [image metadata has been created with Juju][general].
+ - python-openstackclient
+ - python-swiftclient
+
+### Create a Swift container and upload image metatdata
```

Comment:
I know this is really beyond the scope of this change but I think we should generalise this so that it talks about an object store rather than swift. I used these instructions with a swift-free deployment (using ceph rados gateway) and it worked perfectly.

Reply:
Just to be clear, please don't block on this comment. The text is fine as is but it would be nice to generalise the wording at a later date.

Reply:
The upper level heading is now "Upload the Simplestreams Metadata to an object store" and I added a line that swift is used for the example, but any object store can be used.